### DISCUSSION TITLE
Fortnite Festival - Daily Jam Tracks?

### DISCUSSION THREAD

**Discussion by fruitroots** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#248] [Q&A] [answered]
I could only find the endpoint with all available jam tracks but not which ones are active today. Is there no endpoint for this or is it simply not known?

**Comment by LeleDerGrasshalmi** [comment, marked as answer]
there is no own endpoint, its featured jam tracks are based on active event flags
> https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/Fortnite/Calendar.md

they have the following format
> `PilgrimSong.{_title}`

e.g. `PilgrimSong.applause`

**Reply by fruitroots** [reply]
Thanks for the quick reply. Too bad the endpoint is not public and requires authorization.
